### PR TITLE
Company discord channel ID

### PR DESCRIPTION
**Pull request by joaop21**
Add channel_id field to company schema.
Add update function in company_controller, so a manager can change some information about the company.
Change gent.company task, so it can take into account the new field.

Resolves: #144

### CODE REVIEW

**Review by tsource98 — APPROVED** [review]
Seems to be everything fine! 👍

File: lib/safira_web/router.ex [hunk 1]
```diff
@@ -4,7 +4,7 @@ defmodule SafiraWeb.Router do
 
   alias Safira.Guardian
 
-  if Mix.env == :dev do
+  if Mix.env() == :dev do
```
Comment (jpcorreia99) [inline lib/safira_web/router.ex:7]: What is the purpose of this change? Was it not working before

Comment (tsource98) [reply lib/safira_web/router.ex:7]: This change is going to compare directly two atoms instead of getting string.( i guess) 
Check this:
https://elixirforum.com/t/how-to-use-if-or-other-condition-for-dev-or-prod/16235/8

Comment (joaop21) [reply lib/safira_web/router.ex:7]: This change wasn't done by me, I just ran `mix format` against the changed files, and the formatter decided to add this. Nevertheless, @tsource98 is right, just check [this docs](https://hexdocs.pm/mix/Mix.html#env/0).

**Review by jpcorreia99 — APPROVED** [review]
:shipit:
